README.md

```diff
@@ -147,20 +189,30 @@ for i in range(15):
     d.cont()
 ```
 
-To check if a breakpoint was hit by a specific thread, you can use the `hit_on` function:
-
+Since breakpoints in the program text are shared between threads, you can check if a breakpoint was hit on a specific thread with the `hit_on` function:
 ```python
+# Assuming to have thread n.3
+chosen_thread = d.threads[3]
```

Comment:
d.threads[3] does not work yet, d.threads[] is keyed by thread-id.

Reply:
Solved in [740fd13](https://github.com/io-no/libdebug/pull/29/commits/740fd13daa9fbcc76b36aaa7f3725ce30f4933c8)